### PR TITLE
V2

### PR DESCRIPTION
**Pull request by elboletaire**
Among other changes:

- Custom steps (thanks to @hmichal-dev and @jhoover4)
- Custom color bar (thanks to @hmichal-dev and @jhoover4)
- Removed old test environment, replaced with jest.
- Allow setting closest selector (to add compatibility to any css framework, like bootstrap, see #3)
- Other minor fixes (like the "charecters" typo issued in #15 and #27 )

### CODE REVIEW

**Comment by elboletaire** [comment]
@jhoover4 I'd love to see a review from you here. Otherwise I'm publishing this tomorrow, as it passes all the tests and all the related issues have been done already :smile:

**Comment by jhoover4** [comment]
Hey man, at work now but I’ll be sure to check later!

File: .travis.yml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 language: node_js
```
Comment (jhoover4) [inline .travis.yml]: Do we even need travis is you're using gitlab ci now? Should we just remove this?

File: test/password.spec.js [hunk 2]
```diff
@@ -187,6 +184,49 @@ describe('$.fn.password', () => {
...
-    })
+    });
+
+    it('uses no color background image by default', () => {
+      $('#password').password();
+
+      expect('').toEqual($('.pass-colorbar').css('background-image'));
+    });
+
+    it('makes background image style adjustments if turned on', (done) => {
```
Comment (jhoover4) [inline test/password.spec.js]: These `done`s might have been left by me when I merged in and didn't realized we switched to jest. Can we get rid of them?

**Review by jhoover4 — COMMENTED** [review]
Looks good to me besides maybe removing Travis entirely and the test `done`s

File: .travis.yml (hunk 1 above)
Comment (elboletaire) [reply .travis.yml]: I think it can be safely removed, yeah.

**Pull request merged**